### PR TITLE
Add requested modifications

### PR DESCRIPTION
**Pull request by merqurio**
I just added the requested modifications. Thanks guys, first contribution to cask :rocket:

### CODE REVIEW

File: Casks/rodeo.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'rodeo' do
+  version '1.1.19'
+  sha256 '7c91c88715e05cebf6ce2f64551c26d560c1675d786c7f1b5ff6bee03b5fcc79'
+
+  # amazon AWS is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/rodeo.rb]: Should be `# amazonaws.com is the official download host per the vendor homepage`. We’re somewhat strict about the format of the comments.

Comment (vitorgalvao) [inline Casks/rodeo.rb]: I can make that change for you, though. And also squash the commits.

Thank you for contributing!

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 310044421e0172cfc9ba3d45ffb3b108528a24fd. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by merqurio** [comment]
Thank you for the modification, the fast answer and support! I hope I can keep helping making brew cask better :+1:

**Pull request closed without merging**